vsc/model/vsc_generator.py

```diff
@@ -12,14 +12,14 @@
 
 # This performs the following related functions:
 
-# 1. As input.yaml we expect an AST as provided by the parser module
+# 1. As input.yaml we expect an Namespace as provided by the parser module
```

Comment:
I would still say that the whole thing that is provided into the generator is an Abstract Syntax Tree, even if in the current syntax definition the top level node type is Namespace.

I guess in this comment the word AST could be considered the concrete class name, but it could also be considered the "concept" of providing an AST from the parser.  I would personally keep the name AST.

Reply:
I've return AST node back.